integration-cli/docker_cli_run_test.go

```diff
@@ -1353,6 +1353,17 @@ func TestRunWithCpusetCpus(t *testing.T) {
 	logDone("run - cpuset-cpus 0")
 }
 
+func TestRunWithBlkioWeight(t *testing.T) {
```

Comment:
That would be a bit difficult, but doable, the thing is I think it's not necessary.
All Docker needs to be sure is that we write the right value into cgroup files, the functionality is guarantied by cgroup, which is kernel's job. And we don't need to test kernel functionality in Docker. Otherwise, all other resource configs need test to make sure if these resource limit really work. WDYT?